README.md

```diff
@@ -482,6 +486,7 @@ Badge | URL
 ------------ | -------------
 <img src="https://img.shields.io/badge/Adobe%20Dreamweaver-072401?style=for-the-badge&logo=Adobe%20Dreamweaver&logoColor=34F400" />| `https://img.shields.io/badge/Adobe%20Dreamweaver-072401?style=for-the-badge&logo=Adobe%20Dreamweaver&logoColor=34F400`
 <img src="https://img.shields.io/badge/Android_Studio-3DDC84?style=for-the-badge&logo=android-studio&logoColor=white" />| `https://img.shields.io/badge/Android_Studio-3DDC84?style=for-the-badge&logo=android-studio&logoColor=white`
+<img src="https://img.shields.io/badge/apache%20netbeans%20IDE-1B6AC6?style=for-the-badge&logo=apache%20netbeans%20IDE&logoColor=white" /> | `https://img.shields.io/badge/apache%20netbeans%20IDE-1B6AC6?style=for-the-badge&logo=apache%20netbeans%20IDE&logoColor=white`
```

Comment:
It will be `Apache Netbeans`. See [here](https://netbeans.apache.org)

Reply:
> It will be `Apache Netbeans`. See [here](https://netbeans.apache.org)

Thanks for your feedback I will change it